### PR TITLE
JENKINS-64140 fix pom.xml

### PR DESCRIPTION
**Pull request by Absh-Day**
For issue description refer to below:
https://issues.jenkins-ci.org/browse/JENKINS-64140

### CODE REVIEW

File: pom.xml
Comment (oleg-nenashev) [inline pom.xml:3]: Better to use an LTS version as a baseline. Also, this section usually goes after `<packaging>`

File: pom.xml
Comment (oleg-nenashev) [inline pom.xml:10]:
```suggestion
    <version>4.12</version>
```

Comment is obsolete, you might also want to use a recent parent POM release

**Review by oleg-nenashev — COMMENTED** [review]
more fixes are needed to get the plugin up to date, including switching repositories to HTTPs and moving documentation from Wiki to GitHub

**Comment by bromps** [comment]
@oleg-nenashev there is #3 to move the documentation from the wiki to GitHub that I had opened awhile ago. It seems that there is no active maintainer of this repo. I'll volunteer myself to maintain it if no one else is willing to. In the meantime, can we enable issues?

File: pom.xml
Comment (bromps) [reply pom.xml:3]: 2.249.1 should suffice?

Comment (bromps) [reply pom.xml:3]: This build will fail if you don't include `<java.level>8</java.level>` in the `<properties>` block too. The only reason I am suggesting 8 over something newer is because this plugin is pretty old.

**Comment by oleg-nenashev** [comment]
> I'll volunteer myself to maintain it if no one else is willing to

It would be great! Here are the guidelines: https://www.jenkins.io/doc/developer/plugin-governance/adopt-a-plugin/

> In the meantime, can we enable issues?

It is up to the maintainer. Right now Jenkins Jira is used for this plugin

**Comment by Absh-Day** [comment]
@oleg-nenashev correction done!
For the doc part to move from wiki to github, @bromps has already created his PR. 
We need to commit this change so as to unblock other PRs.

**Review by bromps — APPROVED** [review]

**Comment by Absh-Day** [comment]
@bromps Are you maintaining this plugin? Can you help me merge this change?

**Comment by bromps** [comment]
@Absh-Day I just submitted my request for adoption. So, we'll see how that goes. In the meantime, perhaps @oleg-nenashev could triage it for us. I don't know who else to reach out to. Sorry, Oleg! And thanks for all your help.

**Review by MarkEWaite — APPROVED** [review]
Thanks for the contribution and sorry for the delay!

**Pull request merged**
